platform/features/conductor/src/ConductorRepresenter.js

```diff
@@ -97,12 +101,10 @@ define(
             }
 
             function updateDomain(value) {
-                conductor.domain(value);
-                repScope.$broadcast('telemetry:display:bounds', bounds(
-                    conductor.displayStart(),
-                    conductor.displayEnd(),
-                    conductor.domain()
-                ));
+                var newDomain = conductor.domain(value);
+                conductorScope.parameters.format =
+                    newDomain && newDomain.format;
```

Comment:
I had a moment of confusion between "domain object" and "domain object" there :grinning: 

As implemented, no. I was thinking ahead to the case where `TimeConductor` might be populated by metadata from capabilities, but even in that case, I'm thinking now that it should be `TelemetryCapability`'s responsibility to fill in defaults so that user code can just assume complete metadata is available. (I might try to stick those changes into a follow up issue and address only the subset that's needed to simplify this change set.)